checkQC/config.py

```diff
@@ -1,8 +1,30 @@
 
+from pkg_resources import Requirement, resource_filename
+
 import yaml
 
 
 def get_config(config_file):
-    with open(config_file) as stream:
-        config = yaml.load(stream)
-    return config
+    try:
+        if not config_file:
```

Comment:
Very minor: Might want to call it `config_path` or `config_file_name` so it's clear which one is the path and which has the file content.